libbeat/docs/config-file-format.asciidoc

```diff
@@ -0,0 +1,355 @@
+[[config-file-format]]
+== Config file format
+
+Beats config files are based on http://www.yaml.org[YAML], a file format that is
+easier to read and write then other common data formats like XML or JSON.
+
+In beats all YAML files start with a "dictionary" (often called a "hash"). In
```

Comment:
Checking yaml specs I see `dictionary` is not really used. `dictionary` doesn't seem a very good option here. Alternatives to `dictionary` are `object`, `hash` (use by logstash I think, thanks to ruby influence), `table`, `map(-ping)` (yaml spec code docs use `mapping`). For users familiar with JSON, `object` will be most natural.